services/self_hosted.py

```diff
@@ -83,9 +91,15 @@ def license_seats() -> int:
     return license.number_allowed_users or 0
 
 
+async def enterprise_has_seats_left() -> bool:
+    owners = await activated_owners_async()
```

Comment:
Are able to do the owners.count() here directly on the query set?

Reply:
it throws error if I don't make the `count()` sync_to_async as well, the tradeoff is that I get to use the same activated_owner_query that is used elsewhere, but i have to do this additional async step